osu.Game/Screens/Menu/LogoVisualisation.cs

```diff
@@ -34,7 +35,12 @@ public class LogoVisualisation : Drawable, IHasAccentColour
         /// <summary>
         /// The maximum length of each bar in the visualiser. Will be reduced when kiai is not activated.
         /// </summary>
-        private const float bar_length = 600;
+        private const float bar_length_const = 600;
```

Comment:
why rename this?

Reply:
@peppy I had tried having it all in `bar_length`,` but when I tried a map in-game the bottom bar/ timeseeker (if you will) was not rendering as a visualiser, instead they were all at 3 blocks. I made the const so it can work with the amplitude_dead_zone const with no issues